### ISSUE TITLE
Foundations/Setting up Git: Add note about extra github keys

### ISSUE THREAD

**Issue by SumonGFC** [TheOdinProject/curriculum#28716]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

I've noticed some people in the git-help channel on discord asking about what to do if they have already generated and setup an ssh key pair with github on another machine (before starting TOP). I think it would be good to add a lesson note about setting up a new pair of ssh keys. Something to the effect of:

"If you have already setup an ssh key pair with github on a different machine prior to starting the odin project, GitHub allows you to have multiple key pairs associated with your account. You can just follow these instructions again to set up another key pair and register it with GitHub"

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-setting-up-git#step-23-create-an-ssh-key

### (Optional) Discord Name

Chindiana Jones

**Comment by wise-king-sullyman** [comment]
Sounds like a good idea to me, thanks!
